specification/network/resource-manager/Microsoft.Network/stable/2023-09-01/examples/NetworkInterfaceCreate.json

```diff
@@ -9,6 +9,12 @@
         "enableAcceleratedNetworking": true,
         "disableTcpStateTracking": true,
         "ipConfigurations": [
+          {
+            "name": "ipprefixconfig1",
+            "properties": {
+              "privateIPAddressPrefixSize": 28
```

Comment:
>privateIPAddressPrefixSize [](http://example.com/codeflow?start=15&length=26)

Shouldn't this be "privateIPAddressPrefixLength"?

Reply:
Yes, fixed it.